etc/docker/.env.execution.dev

```
@@ -10,15 +10,19 @@ ASPNETCORE_ENVIRONMENT=Development
 DAPR_PUBSUB_STORE_NAME=vnext-execution-pubsub
+DAPR_PUBSUB_BROADCAST_STORE_NAME=vnext-pubsub-broadcast
 DAPR_LOCK_STORE_NAME=vnext-execution-lock
 OTEL_EXPORTER_OTLP_ENDPOINT=http://otel-collector:4317
 OTEL_SERVICE_NAME=vnext-execution-app
 ConnectionStrings__Default=Host=vnext-postgres;Port=5432;Database=Aether_WorkflowDb;Username=postgres;Password=postgres;
 Redis__Standalone__EndPoints__0=vnext-redis:6379
-Telemetry__Otlp__Endpoint=http://otel-collector:4317
-vNextApi__BaseUrl:http://vnext-app:5000
\ No newline at end of file
+Vault__Enabled=false
+vNextApi__BaseUrl:http://vnext-app:5000
```

dotenv-linter (4.0.0)

[warning] [IncorrectDelimiter] The vNextApi__BaseUrl:http://vnext-app:5000 key has incorrect delimiter

(IncorrectDelimiter)